Samples/word-citation-management/manifest.json

```diff
@@ -0,0 +1,215 @@
+    "authorization": {
+        "permissions": {
+            "resourceSpecific": [
+                {
+                    "name": "Document.ReadWrite.User",
+                    "type": "Delegated"
+                }
+            ]
+        }
+    },
+    "extensions": [
+        {
+            "requirements": {
+                "scopes": [
+                    "document"
+                ]
+            },
+            "runtimes": [
+                {
+                    "requirements": {
+                        "capabilities": [
+                            {
+                                "name": "Taskpane",
+                                "minVersion": "1.1"
+                            }
+                        ],
+                        "scopes": [
+                            "document"
+                        ],
+                        "formFactors": [
+                            "desktop"
+                        ]
+                    },
+                    "id": "runtime_1",
+                    "type": "general",
+                    "code": {
+                        "page": "https://localhost:3000/taskpane.html",
+                        "script": "https://localhost:3000/taskpane.js"
+                    },
+                    "lifetime": "short"
+                },
+                {
+                    "requirements": {
+                        "capabilities": [
+                            {
+                                "name": "AddinCommands",
+                                "minVersion": "1.1"
+                            }
+                        ]
+                    },
+                    "id": "runtime_2",
+                    "type": "general",
+                    "code": {
+                        "page": "https://localhost:3000/taskpane.html",
+                        "script": "https://localhost:3000/taskpane.js"
+                    },
+                    "lifetime": "short",
+                    "actions": [
+                        {
+                            "id": "ButtonId1_1",
+                            "type": "openPage",
+                            "pinnable": false,
+                            "view": "ButtonId1",
+                            "multiselect": false,
+                            "supportsNoItemContext": false
+                        }
+                    ]
+                }
+            ],
+            "getStartedMessages": [
+                {
+                    "requirements": {
+                        "capabilities": [
+                            {
+                                "name": "Taskpane",
+                                "minVersion": "1.1"
+                            }
+                        ],
+                        "scopes": [
+                            "document"
+                        ],
+                        "formFactors": [
+                            "desktop"
+                        ]
+                    },
```

Comment:
This "requirements" property is a mistake by the converter executable. Delete it.